### PR TITLE
[MGDAPI-3557] ci: add prow checks for MGDAPI-2427

### PR DESCRIPTION
**Pull request by cecobask**
Adding prow checks for branch https://github.com/integr8ly/cloud-resource-operator/tree/MGDAPI-2427

### CODE REVIEW

File: ci-operator/config/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-MGDAPI-2427.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+- dockerfile_path: build/Dockerfile
+  from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/integr8ly/cloud-resource-operator/build
+  to: cloud-resource-operator
+promotion:
+  name: MGDAPI-2427
```
[the remark this answers is not in the document]
Comment (laurafitzgerald) [reply ci-operator/config/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-MGDAPI-2427.yaml:23]: I think was an issue with building images. Is it an issue for cro?

[the remark this answers is not in the document]
Comment (cecobask) [reply ci-operator/config/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-MGDAPI-2427.yaml:23]: I read about the lowercase branch name, but I saw that other config aren't in lowercase. Example: https://github.com/openshift/release/blob/master/ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-MGDAPI-1355-1.yaml

[the remark this answers is not in the document]
Comment (austincunningham) [reply ci-operator/config/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-MGDAPI-2427.yaml:23]: @laurafitzgerald not sure to be honest, we have never had a CRO feature branch don't see a plugin config for CRO either so may need to add one for CRO https://github.com/openshift/release/pull/26497/files#diff-a9035bbaee5af66130c81cfd0b97df589a7d2e22e87ad9d3800851407967c07bR738

**Pull request merged**